OpenRA.Mods.Common/Traits/Demolishable.cs

```diff
@@ -17,11 +17,26 @@ public class DemolishableInfo : IDemolishableInfo, ITraitInfo
 	{
 		public bool IsValidTarget(ActorInfo actorInfo, Actor saboteur) { return true; }
 
-		public object Create(ActorInitializer init) { return new Demolishable(); }
+		[Desc("If true and this actor has EjectOnDeath, no actor will be spawned.")]
+		public readonly bool PreventsEjectOnDeath = true;
```

Comment:
Units spawning from demolished structures was an important (and frustrating) part of the commando missions in TD.